src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
I thought about it, but it would be to a lot of duplication, as the implementation would be the same for osx/freebsd and all the linux variants.
Also it was implemented this way in src/process/executable_path.cr

Should I still move it there ?